foghorn-edge/templates/cleanup-cluster-role.yaml

```diff
@@ -0,0 +1,41 @@
+kind: ClusterRole
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.